src/components/LinkButton/LinkButton.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from 'clsx';
+import Link from 'next/link';
+import React from 'react';
+import styles from './LinkButton.module.css';
+
+interface ButtonProps {
+  href: string;
+  label: string;
+  variant: 'green' | 'gray';
+}
+
+export const LinkButton: React.FC<ButtonProps> = ({ href, label, variant }) => {
+  const className = clsx(
+    styles.button,
+    variant === 'green' && styles.greenButton,
+    variant === 'gray' && styles.grayButton,
+  );
+
+  return (
+    <Link href={href} className={className}>
+      {label}
+    </Link>
+  );
+};
```

Comment:
```suggestion
interface ButtonProps {
  href: string;
  variant: 'green' | 'gray';
  children: React.ReactNode;
}

export const LinkButton: React.FC<ButtonProps> = ({ href, variant, children }) => {
  const className = clsx(
    styles.button,
    variant === 'green' && styles.greenButton,
    variant === 'gray' && styles.grayButton,
  );
  
  return (
    <Link href={href} className={className}>
      {children}
    </Link>
  );
```



今のままだと
```tsx
<LinkButton href='/foo' variant='green' title='ここをクリック'/>
```

みたいな使い方になってしまうから、

```tsx
<LinkButton href='/foo' variant='green''>
    ここをクリック
</LinkButton>
```

って感じで使えるように修正しよう！